.vscode/launch.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
Please remove this directory as we don't want local configurations in the repository.